download.sh

```diff
@@ -8,7 +8,7 @@ set -u
 set -o pipefail
 
 readonly ROOT=".rally/benchmarks"
-readonly URL="http://benchmarks.elasticsearch.org.s3.amazonaws.com/corpora"
+readonly URL="http://rally-tracks.elastic.co"
```

Comment:
This only works with HTTPS for me:

```suggestion
readonly URL="https://rally-tracks.elastic.co"
```

Reply:
Good catch